src/hashmap.c

```diff
@@ -99,21 +95,27 @@ bool hash_RemoveElement(HashMap map, char const *key)
 	return false;
 }
 
-void *hash_GetElement(HashMap const map, char const *key)
+void **hash_GetNode(HashMap const map, char const *key)
 {
 	HashType hashedKey = hash(key);
 	struct HashMapEntry *ptr = map[(HalfHashType)hashedKey];
 
 	while (ptr) {
 		if (hashedKey >> HALF_HASH_NB_BITS == ptr->hash
 		 && !strcmp(ptr->key, key)) {
-			return ptr->content;
+			return &ptr->content;
 		}
 		ptr = ptr->next;
 	}
 	return NULL;
 }
 
+void *hash_GetElement(HashMap const map, char const *key)
+{
+	void **node = hash_GetNode(map, key);
+	return node ? *node : NULL;
```

Comment:
Again, [checkpatch requires a blank line](https://github.com/gbdev/rgbds/pull/833/checks#step:4:38). (Ideally, please also fix the other reports.)

Reply:
Ah, sorry about that. I somehow reintroduced it during editing. All issues fixed.